pkg/instrumentation/defaultinstrumentation.go

```diff
@@ -19,34 +20,11 @@ const (
 	defaultNamespace      = "default"
 	defaultKind           = "Instrumentation"
 
-	otelSampleEnabledKey                       = "OTEL_SMP_ENABLED" //TODO: remove in favor of new name once safe
```

Comment:
Are we suppose to remove all these const?

Reply:
yeah I removed the consts and just put them directly in the Instrumentation struct for better readability